codegen_java/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.10.2
```

Comment:
of note: I did not change the `sbt.version` in the codegen_java_old directory. Why do we have an "old" directory committed to source control?